bin/apisix

```diff
@@ -450,6 +450,7 @@ init:       initialize the local nginx.conf
 init_etcd:  initialize the data of etcd
 start:      start the apisix server
 stop:       stop the apisix server
+restart:    stop the apisix server and then start the apisix server
```

Comment:
`restart the apisix server` would be better.

Reply:
okay